src/microbe_stage/CompoundCloudPlane.tscn

```diff
@@ -1,26 +1,37 @@
-[gd_scene load_steps=7 format=3 uid="uid://bertyonikiucy"]
+[gd_scene load_steps=8 format=3 uid="uid://bertyonikiucy"]
 
 [ext_resource type="Shader" path="res://shaders/CompoundCloudPlane.gdshader" id="1"]
-[ext_resource type="Texture2D" uid="uid://xj4714l7pucx" path="res://assets/textures/PerlinNoise.jpg" id="2"]
 [ext_resource type="Script" path="res://src/microbe_stage/CompoundCloudPlane.cs" id="3"]
-[ext_resource type="Texture2D" uid="uid://bd6mlts55wpaf" path="res://assets/textures/TestCloudConcentrations.png" id="4"]
 
 [sub_resource type="QuadMesh" id="1"]
+lightmap_size_hint = Vector2i(1002, 1002)
 size = Vector2(200, 200)
 
+[sub_resource type="Gradient" id="Gradient_u1fse"]
+offsets = PackedFloat32Array(0, 0.88806)
+colors = PackedColorArray(0.385493, 0.385492, 0.385492, 1, 1, 1, 1, 1)
+
+[sub_resource type="FastNoiseLite" id="FastNoiseLite_7cqxm"]
+
+[sub_resource type="NoiseTexture2D" id="NoiseTexture2D_xdmng"]
+generate_mipmaps = false
+seamless = true
+color_ramp = SubResource("Gradient_u1fse")
+noise = SubResource("FastNoiseLite_7cqxm")
+
 [sub_resource type="ShaderMaterial" id="2"]
 resource_local_to_scene = true
 render_priority = -1
 shader = ExtResource("1")
-shader_parameter/colour1 = Color(0, 0, 0, 0)
-shader_parameter/colour2 = Color(0, 0, 0, 0)
-shader_parameter/colour3 = Color(0, 0, 0, 0)
-shader_parameter/colour4 = Color(0, 0, 0, 0)
+shader_parameter/colour1 = Color(1, 1, 0, 1)
+shader_parameter/colour2 = Color(0, 0, 1, 1)
+shader_parameter/colour3 = Color(0, 1, 0, 1)
+shader_parameter/colour4 = Color(1, 0, 0, 1)
 shader_parameter/UVOffset = Vector2(0, 0)
-shader_parameter/BrightnessMultiplier = 1.0
-shader_parameter/CloudAlpha = 0.7
-shader_parameter/densities = ExtResource("4")
-shader_parameter/noise = ExtResource("2")
+shader_parameter/NoiseScale = 0.05
+shader_parameter/BrightnessMultiplier = 2.0
```

Comment:
If brightness multipliers are intended to be changed the biomes.json file also needs to change. Also the default colours above, I think they should stay black transparent colour as unused channels should default to that value to not accidentally display.

Reply:
oh if you're referring to this part of code i think it was a typo, i restored the original brightness and it should remain as it was initially

Reply:
The brightness was actually intentionally set to 2 there, as at 1 with the new shader they looked desaturated to me. The brightness values would have to be tested the patches, but they are multipliers, so if it looks fine it should be fine.

Reply:
I guess we can have 2 follow up PRs then (the first one will be the changed absorbing logic).